vshard.go

```diff
@@ -127,6 +127,10 @@ type InstanceInfo struct {
 	UUID uuid.UUID
 }
 
+func (ii InstanceInfo) String() string {
+	return fmt.Sprintf("{name: %s, uuid: %s, addr: %s}", ii.Name, ii.UUID, ii.Addr)
```

Comment:
I guess this Srting() method is not required, because it's almost the same as
```golang
func (ii InstanceInfo) String() string {
	type alias InstanceInfo
	return fmt.Sprintf("%+v", alias(ii))
}
```

It means we can just use `%+v` formatter instead of `%v`

Reply:
yes, we can, but it becomes less readable and clearer
I think this ormat is more reliable